<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>xVSaveDlg</class>
 <widget class="QDialog" name="xVSaveDlg">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>373</width>
    <height>423</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>save dash board</string>
  </property>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>6</x>
     <y>6</y>
     <width>361</width>
     <height>411</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QLabel" name="pPixLab">
      <property name="sizePolicy">
       <sizepolicy hsizetype="MinimumExpanding" vsizetype="MinimumExpanding">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="frameShape">
       <enum>QFrame::Box</enum>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
    </item>
    <item>
     <layout class="QGridLayout" name="gridLayout">
      <item row="1" column="0" colspan="2">
       <widget class="QLabel" name="label_3">
        <property name="text">
         <string>created at</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
        </property>
       </widget>
      </item>
      <item row="2" column="0" colspan="2">
       <widget class="QLabel" name="label_4">
        <property name="text">
         <string>description</string>
        </property>
       </widget>
      </item>
      <item row="0" column="0" colspan="2">
       <widget class="QLabel" name="label">
        <property name="text">
         <string>created by</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
        </property>
       </widget>
      </item>
      <item row="3" column="2">
       <widget class="QLineEdit" name="pFolderLEdit">
        <property name="readOnly">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item row="3" column="3">
       <widget class="QToolButton" name="pBrowseTB">
        <property name="text">
         <string>...</string>
        </property>
       </widget>
      </item>
      <item row="2" column="2" colspan="2">
       <widget class="QLineEdit" name="pDescLEdit">
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
      <item row="1" column="2" colspan="2">
       <widget class="QDateTimeEdit" name="dateTimeEdit">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="displayFormat">
         <string>hh:mm:ss dd.MMMM.yyyy</string>
        </property>
       </widget>
      </item>
      <item row="0" column="2" colspan="2">
       <widget class="QLineEdit" name="pCreatedByLEdit">
        <property name="text">
         <string>anonymous</string>
        </property>
       </widget>
      </item>
      <item row="3" column="0" colspan="2">
       <widget class="QLabel" name="label_2">
        <property name="text">
         <string>folder name</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout"/>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <item>
       <widget class="QCheckBox" name="pIncludeDataCB">
        <property name="text">
         <string>include data</string>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="horizontalSpacer">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QToolButton" name="pCancelTB">
        <property name="text">
         <string>&amp;cancel</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QToolButton" name="pSaveTB">
        <property name="text">
         <string>&amp;save</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
 </widget>
 <tabstops>
  <tabstop>pCreatedByLEdit</tabstop>
  <tabstop>dateTimeEdit</tabstop>
  <tabstop>pDescLEdit</tabstop>
  <tabstop>pFolderLEdit</tabstop>
  <tabstop>pBrowseTB</tabstop>
  <tabstop>pIncludeDataCB</tabstop>
  <tabstop>pCancelTB</tabstop>
  <tabstop>pSaveTB</tabstop>
 </tabstops>
 <resources/>
 <connections>
  <connection>
   <sender>pCancelTB</sender>
   <signal>clicked()</signal>
   <receiver>xVSaveDlg</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>319</x>
     <y>414</y>
    </hint>
    <hint type="destinationlabel">
     <x>116</x>
     <y>199</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pSaveTB</sender>
   <signal>clicked()</signal>
   <receiver>xVSaveDlg</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>364</x>
     <y>414</y>
    </hint>
    <hint type="destinationlabel">
     <x>136</x>
     <y>189</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
